code/modules/psionics/faculties/coercion.dm

```diff
@@ -60,6 +60,21 @@
 		to_chat(user, SPAN_WARNING("\The [target] is in no state for a mind-read."))
 		return TRUE
 
+	var/obj/item/organ/internal/augment/psi/psiaug
+	for (psiaug in target)
+		if(!psiaug.is_broken())
+			break
+	if (!psiaug)
+		if(target.isSynthetic())
+			to_chat(user, SPAN_ALIEN("There is no mind here for you to dip your mentality into."))
+			return
```

Comment:
not sold on psiaugs allowing people to have their mind read

Reply:
This is to stop it from normally working on races without a zona bovinae/brain.